yaptide/application.py

```diff
@@ -17,6 +17,15 @@ def create_app(config_object="yaptide.settings"):
 
     api = Api(app)
     initialize_routes(api)
+
+    @app.after_request
+    def after_request(response):
+        response.headers.add('Access-Control-Allow-Origin', '*')
```

Comment:
Why this is needed?

Reply:
This solves the problem with CORS, which occured during connecting back with front

Reply:
add a comment with detailed explanation